claude-md-improvements.md

```
@@ -0,0 +1,219 @@
+   # Verify build artifacts are in expected locations
+   yarn build
+   find . -name "ReactOnRails.full.js" -type f
+   ```
+````
+
+4. **Use variables/constants when possible:**
+   - In JavaScript/TypeScript, use `__dirname` or import.meta.url
+   - In Ruby, use `Rails.root` or `File.expand_path`
+   - Avoid hardcoding relative paths in config files when possible
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)